.github/workflows/windows.yml

```diff
@@ -338,29 +194,23 @@ jobs:
       - name: Download OpenVINO package
         uses: actions/download-artifact@v4
         with:
-          name: openvino_package
+          name: ${{ env.OV_ARTIFACT_NAME }}
           path: ${{ env.INSTALL_DIR }}
 
-      - name: Extract OpenVINO packages
-        run: |
-          pushd ${{ env.INSTALL_DIR }}
-            Expand-Archive openvino_package.zip -DestinationPath ./tmp
-            mv ./tmp/*/* .
-          popd
-
       #
       # Build
       #
 
       - name: Build tokenizers wheel
         run: |
           python -m pip wheel -v --no-deps --wheel-dir ${env:BUILD_DIR} `
-            --find-links ${env:INSTALL_DIR}/tools `
+            ${{ needs.openvino_download.outputs.ov_wheel_source }} `
             --config-settings=override=cmake.build_path="${env:CMAKE_BUILD_DIR}" `
             ${env:OPENVINO_TOKENIZERS_REPO}
         env:
           CMAKE_BUILD_PARALLEL_LEVEL: '4'
           CMAKE_BUILD_DIR: 'D:\a\b'
+        working-directory: ${{ env.INSTALL_DIR }}
```

Comment:
it will have gone once openvino provider can extract artifacts to a given path, right?